respa/settings.py

```diff
@@ -156,6 +157,7 @@
                 'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
                 'django.contrib.messages.context_processors.messages',
+                'respa_admin.context_processors.export_global_vars',
```

Comment:
Instead of a context processor (which affects all template rendering in the system, not just respa admin), this could alternatively be implemented as a Mixin class which is inherited in the Respa admin views.

Something like
```
class ExtraContextMixin():
    def get_context_data(self, **kwargs):
        context = super().get_context_data(**kwargs)
        context['INSTRUCTIONS_URL'] = settings.FOO
        return context
```

Then this could be used in Respa admin views like: `class SaveResourceView(ExtraContextMixin, CreateView):`

This way the data would only be injected to respa admin template contexts. I'm not saying this has to be changed, but if the global context processor is used, I'd use a more specific context variable name than `INSTRUCTIONS_URL`, maybe prefix like `RESPA_ADMIN_INSTRUCTIONS_URL`.